src/DataTransferObject.php

```diff
@@ -390,4 +407,20 @@ private function recursiveToArray(array $data): array
 
         return $data;
     }
+
+    /**
+     * @return array
+     */
+    public function getUnknownProperties(): array
+    {
+        return $this->unknownProperties;
+    }
+
+    /**
+     * @param array $unknownProperties
+     */
+    public function setUnknownProperties(array $unknownProperties): void
+    {
+        $this->unknownProperties = $unknownProperties;
+    }
```

Comment:
I'm a bit iffy on this being public but understand why it is.

Reply:
Yeah I can move it in version 1.0.0 when we're allowed to break contracts. Putting it in the constructor would break every manually created DTO (which is none).

Reply:
It's been removed downstream so all good.